### DISCUSSION TITLE
Update Index files

### DISCUSSION THREAD

**Discussion by T-prog3** [TumblThreeApp/TumblThree#276] [Ideas]
I just moved my download folder to a new disc and then tried to update the download locations in `General` and in all `Collections`.
I was thinking of `download location` as "_Point me to the folder that have an Index folder with .twitter files in it and if not i create one._" So i thought TumblThree would find it with just a simple switch. But that didn't work, so i had to replace the path at `ChildId":` and `Location":` in all Index files to fix it manually.

I think that when a user move their files and change the `download location` of a `Collection` then the Index files in that `Collection` should be updated with our new location at `ChildId":` in **.twitter** and  `Location":` in both **.twitter** and **_files.twitter** automatic. 
As of now there's an error that break collections.

**Comment by thomas694** [comment]
Thanks for your feature suggestion. We'll think about it.
Originally TumblThree is not made for recognizing moved download folders, but maybe it could recognize it and ask the user, if the blog files shall be adjusted.

In the meantime collections have been introduced. I think, it should have been possible to create a new collection on the new disk and assign this new collection to the desired blogs. This would have moved the blog index files correctly and afterwards the blog data folders would have needed to be moved to the new location.
But (to prevent chaos) there is no functionality to change the collection of multiple blogs at once.

> As of now there's an error that break collections.

Do you consider your desired behavior an error, or is there another (additional) error?

**Comment by T-prog3** [comment]
**Let me clarify:**
1. Lets say you have a collection named myCollection with a `Download Location: C:\TumblThree\myCollectionFolder` to save all their blogs and data in.
2. Now you move the folder `myCollectionFolder` into `D:\Tumblr\`.
3. Then you change your download location of that collection to `D:\Tumblr\myCollectionFolder` and save.
4. Restart TumblThree and then switch to that collection.
5. Now you got an error message showing for each blog in that list.
6. TumblThree knows about all blogs but doesn't have enough information to load the blogs into the collection.
7. Solution was to manually update `ChildId: and Location:`

**Desired behavior:**
**1)**
If TumblThree recognize the blogs then there should at least be enough information to load all blogs to my collection even if it still downloads all new files to the old location.

**2)** 
- If i have a specified "Download Location:" of a collection then i would like to have an update function running when you change this location.  
 - This should localizes the **.twitter files** in our old/new locations Index folder and update if there's blogs in it. 
 - Showing something like a dialog prompt `Do you want to update download location: %Location% of blog: %Blog%. [Yes] [Yes to All] [No] [No to All]`
